src/utils/createRequire.js

```diff
@@ -0,0 +1,4 @@
+export default (module, disableProcessing) => {
+  const loader = disableProcessing ? '-!' + module : module;
```

Comment:
Yak shaving, voodoo coding. I was searching for what is causing increased build time (it's not it, see details above). Actually I took it from previous version of `bootstrap-sass-loader`. If it's for no reason there, I'll remove it from here.